modules/decima-ext-model-exporter/src/main/java/com/shade/decima/ui/data/viewer/model/dmf/DMFExporter.java

```diff
@@ -1356,7 +1541,14 @@ private DMFTexture exportTexture(@NotNull RTTIObject texture, @NotNull String te
             return null;
         }
         final ByteArrayOutputStream stream = new ByteArrayOutputStream();
-        new TextureExporterPNG().export(imageProvider, Set.of(), Channels.newChannel(stream));
+        final String ext;
+        if (imageProvider.getBitsPerChannel() > 1) {
+            new TextureExporterTIFF().export(imageProvider, Set.of(), Channels.newChannel(stream));
+            ext = ".tiff";
+        } else {
+            new TextureExporterPNG().export(imageProvider, Set.of(), Channels.newChannel(stream));
+            ext = ".png";
+        }
```

Comment:
It's quite a weird way to determine a suitable output container. Can't you stick with DDS and use your texture library if needed?

Reply:
DDS isn't fully supported in the blender